src/components/ListItemIcon/ListItemIcon.vue

```diff
@@ -57,7 +57,7 @@ It might be used for list rendering or within the multiselect for example
 </docs>
 
 <template>
-	<span class="option" :style="cssVars">
+	<span :id="id" class="option" :style="cssVars">
```

Comment:
```suggestion
	<span id="" class="option" :style="cssVars">
```
Wouldn't this be enough then? Or do we need the possibility to bind an ID there?

Reply:
I like the idea of keeping the id possible. Especially on lists where you can scroll to the element with the ID hash

Reply:
Fine with me.